packages/uikit/src/widgets/Menu/components/footerConfig.ts

```diff
@@ -7,19 +7,19 @@ export const footerLinks: (t: ContextApi["t"]) => FooterLinkType[] = (t) => [
     items: [
       {
         label: t("Trade"),
-        href: "/swap",
+        href: "https://pancakeswap.finance/swap",
```

Comment:
We should still use relative path for swap and farms?

Reply:
This will cause 404 at `https://pancakeswap.games/`
at Aptos `/nfts` & `/v4` will also return 404

Reply:
Can we have different footer links configured for different apps? We can have some common links but I think essentially they should have different links. Otherwise the swap and farm link will be redirecting to the evm site for aptos

Reply:
Ok. Will make a config only for Aptos. Other will maintain with this

Reply:
done